near-sdk/src/utils/mod.rs

```diff
@@ -51,6 +51,11 @@ pub fn assert_one_yocto() {
     assert_eq!(env::attached_deposit(), 1, "Requires attached deposit of exactly 1 yoctoNEAR")
 }
 
+/// Assert that at least 1 yoctoNEAR was attached.
+pub fn assert_at_least_one_yocto() {
```

Comment:
So one note about this is that we would need to bump the SDK version to be compatible with the NFT standard. If we want to make this accessible ASAP to people using the current version of the SDK, maybe it's worth either redefining in the contract-standards for now or just calling the assert directly?

Also just unclear the greater usage of this, we might want to be careful before adding another public function as #380 was also confusing to me.